### PR TITLE
add precourse

### PR DESCRIPTION
**Pull request by sergisanrama**

### CODE REVIEW

File: staff/sergi-sanrama/precourse/tema3-functions/project3.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
+function bingo(){
+
+function userName(){
```
Comment (ghost) [inline staff/sergi-sanrama/precourse/tema3-functions/project3.js:3]: indent !

File: staff/sergi-sanrama/precourse/tema3-functions/project3.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
+function bingo(){
+
+function userName(){ 
+  var person = prompt("Can you tell us your name?"); 
+    if (person) { 
+      return person;
+    } else { (person != typeof(string))
```
Comment (ghost) [inline staff/sergi-sanrama/precourse/tema3-functions/project3.js:7]: wtf ?

File: staff/sergi-sanrama/precourse/tema4-challenges/project4.js [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+function gameStart(){
+    hello();
+    allQuestions(questions);
+    repeat(questions);
+    byeBye();
+    playAgain();
+};
+
+function userName(){
```
Comment (ghost) [inline staff/sergi-sanrama/precourse/tema4-challenges/project4.js:43]: very good !

**Review by carlotavicens — COMMENTED** [review]
en general falten bastants semicolon

**Comment by alejaa03** [comment]
.

File: staff/sergi-sanrama/precourse/tema3-functions/project3.js (hunk 1 above)
Comment (sergisanrama) [reply staff/sergi-sanrama/precourse/tema3-functions/project3.js:3]: accomplished my friend

File: staff/sergi-sanrama/precourse/tema3-functions/project3.js (hunk 2 above)
Comment (manuelbarzi) [reply staff/sergi-sanrama/precourse/tema3-functions/project3.js:7]: WHAT are you trying to resolve with that ```typeof```? do you understand ```typeof```? please, take a minute and RTFM => https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/typeof

Comment (sergisanrama) [reply staff/sergi-sanrama/precourse/tema3-functions/project3.js:7]: The type here does not make sense, fixed, thnks!

**Pull request merged**